### DISCUSSION TITLE
LibDualSpec Bug

### DISCUSSION THREAD

**Discussion by Jinnai** [SFX-WoW/Masque#219] [Issues] [answered]
LibDualSpec gives an error because `WOW_PROJECT_ID == 5` on TBC Classic. I suggest instead

`if WOW_PROJECT_ID ~= WOW_PROJECT_MAINLINE then return end`

Edit: just saw the lib is already updated, just need to bundle the new version for TBC

**Comment by StormFX** [comment, marked as answer]
Thanks for the report (and update). This is one of those issues that require handling by the library author(s). Masque doesn't include LDS in the Classic version, which is the one that should be used for TBC. That said, I do need to update a check in my code, but it shouldn't break anything for now. I'll fix it when TBC-Classic is released.

Closing.
